[2024-02-28T03:29:31.632+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_forecast_pipeline.load_to_database scheduled__2024-01-01T06:00:00+00:00 [queued]>
[2024-02-28T03:29:31.676+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_forecast_pipeline.load_to_database scheduled__2024-01-01T06:00:00+00:00 [queued]>
[2024-02-28T03:29:31.682+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-28T03:29:31.715+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_database> on 2024-01-01 06:00:00+00:00
[2024-02-28T03:29:31.731+0000] {standard_task_runner.py:60} INFO - Started process 1726 to run task
[2024-02-28T03:29:31.747+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sales_forecast_pipeline', 'load_to_database', 'scheduled__2024-01-01T06:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/sales_forecast_pipeline.py', '--cfg-path', '/tmp/tmprzwd8pnh']
[2024-02-28T03:29:31.748+0000] {standard_task_runner.py:88} INFO - Job 48: Subtask load_to_database
[2024-02-28T03:29:31.874+0000] {task_command.py:423} INFO - Running <TaskInstance: sales_forecast_pipeline.load_to_database scheduled__2024-01-01T06:00:00+00:00 [running]> on host 15313af250a4
[2024-02-28T03:29:32.033+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sales_forecast_pipeline' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T06:00:00+00:00'
[2024-02-28T03:29:34.097+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-28T03:29:34.107+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sales_forecast_pipeline, task_id=load_to_database, execution_date=20240101T060000, start_date=20240228T032931, end_date=20240228T032934
[2024-02-28T03:29:34.149+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-28T03:29:34.176+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
